### ISSUE TITLE
Navigation to certain URLs report the requests as canceled in devtools.

### ISSUE THREAD

**Issue by Saurbaum** [cefsharp/CefSharp#2036]
- **What version of the product are you using?**
Using nuget 53.0.0 (last known working version 51.0.0)

- **What architecture x86 or x64?**
x86

- **On what operating system?**
Windows 7

- **Are you using `WinForms`, `WPF` or `OffScreen`?**
WPF

- **What steps will reproduce the problem?**
Host the CEFSharp browser in a window and then in code behind set it to load the url
```
this.browser.Load("https://idpgis.ncep.noaa.gov/arcgis/rest/services/NWS_Observations/radar_base_reflectivity/MapServer/export?F=image&FORMAT=PNG32&TRANSPARENT=true&SIZE=256%2C256&BBOX=-11271098.442818949%2C3757032.814272983%2C-10018754.171394622%2C5009377.085697311&BBOXSR=3857&IMAGESR=3857&DPI=90");
```

- **What is the expected output? What do you see instead?**
This should load a section of a map as a png detailing cloud formations over North America.  If you paste the url into chrome, IE or CEFSharp 51.0.0 or earlier that is what you will see.
Instead if you look in the developer tools the network request will be marked as canceled.

- **Please provide any additional information below.**
```
{
  "log": {
    "version": "1.2",
    "creator": {
      "name": "WebInspector",
      "version": "537.36"
    },
    "pages": [],
    "entries": [
      {
        "startedDateTime": "2017-05-03T08:50:08.078Z",
        "time": 0,
        "request": {
          "method": "GET",
          "url": "https://idpgis.ncep.noaa.gov/arcgis/rest/services/NWS_Observations/radar_base_reflectivity/MapServer/export?F=image&FORMAT=PNG32&TRANSPARENT=true&SIZE=256%2C256&BBOX=-11271098.442818949%2C3757032.814272983%2C-10018754.171394622%2C5009377.085697311&BBOXSR=3857&IMAGESR=3857&DPI=90",
          "httpVersion": "unknown",
          "headers": [
            {
              "name": "Upgrade-Insecure-Requests",
              "value": "1"
            },
            {
              "name": "User-Agent",
              "value": "Mozilla/5.0 (Windows NT 6.1; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/53.0.2785.116 Safari/537.36"
            }
          ],
          "queryString": [
            {
              "name": "F",
              "value": "image"
            },
            {
              "name": "FORMAT",
              "value": "PNG32"
            },
            {
              "name": "TRANSPARENT",
              "value": "true"
            },
            {
              "name": "SIZE",
              "value": "256%2C256"
            },
            {
              "name": "BBOX",
              "value": "-11271098.442818949%2C3757032.814272983%2C-10018754.171394622%2C5009377.085697311"
            },
            {
              "name": "BBOXSR",
              "value": "3857"
            },
            {
              "name": "IMAGESR",
              "value": "3857"
            },
            {
              "name": "DPI",
              "value": "90"
            }
          ],
          "cookies": [],
          "headersSize": -1,
          "bodySize": 0
        },
        "response": {
          "status": 0,
          "statusText": "",
          "httpVersion": "unknown",
          "headers": [],
          "cookies": [],
          "content": {
            "size": 0,
            "mimeType": "x-unknown"
          },
          "redirectURL": "",
          "headersSize": -1,
          "bodySize": -1,
          "_transferSize": 0,
          "_error": ""
        },
        "cache": {},
        "timings": {
          "blocked": -1,
          "dns": -1,
          "connect": -1,
          "send": 0,
          "wait": 0,
          "receive": 0,
          "ssl": -1
        },
        "serverIPAddress": ""
      }
    ]
  }
}
```

- **Does the cef log provide any relevant information? (By default there should be a debug.log file in your bin directory)**
No

- **Any other background information that's relevant? Are you doing something out of the ordinary? 3rd party controls?**
No

- **Does this problem also occur in the `CEF` Sample Application from http://opensource.spotify.com/cefbuilds/index.html?**
No

**Comment by amaitland** [comment]
> Using nuget 53.0.0 (last known working version 51.0.0)

`53.0.0` is old and no longer supported. It also has `Known issues` with `SSL` certificates being rejected, which is listed in the **Known Issues** section of the release notes. You need to upgrade, the latest version is `57.0.0`.

Please show some patience, you asked your question on `Gitter`, give it at least a couple of days for someone to reply.

**Comment by Saurbaum** [comment]
I'm sorry I should have been clearer every version post 51 doesn't work. I
meant to say that it worked in 51 and hasn't since.

Now is that same issue still affecting 57 because the problem still exists
in it?

On 4 May 2017 00:48, "Alex Maitland" <notifications@github.com> wrote:

Using nuget 53.0.0 (last known working version 51.0.0)

53.0.0 is old and no longer supported. It also has Known issues with SSL
certificates being rejected, which is listed in the *Known Issues* section
of the release notes. You need to upgrade, the latest version is 57.0.0.

Please show some patience, you asked your question on Gitter, give it at
least a couple of days for someone to reply.

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
<https://github.com/cefsharp/CefSharp/issues/2036#issuecomment-299066475>,
or mute the thread

**Comment by amaitland** [comment]
> Now is that same issue still affecting 57 because the problem still exists in it?

https://github.com/cefsharp/CefSharp/blob/master/CONTRIBUTING.md#cefsharp-vs-chromium-embedded-frameworkcef

**Comment by Saurbaum** [comment]
I've checked it with the example chromium build and there was no issue as I indicated in the bug report so while it appears to be low level there is a clear difference between the two.

I've read the linked article again. And once again I'm sorry for not being clearer in the first place but now are you asking me for a sample or to attempt a fix myself or indicating that it is fixed already in a later version of chromium that CEFSharp hasn't moved to yet?
